assets/scss/modules/_forms.scss

```diff
@@ -16,14 +17,14 @@
   }
 }
 
-.content-checkbox {
+.checkbox-group {
   width: 100%;
   position: relative;
   display: inline-block;
   text-align: left;
   margin: 20px 0;
 
-  .checkbox {
+  .holder-checkbox {
```

Comment:
I see there is a difference in the way you made this clases `.checkbox-group` and `.holder-checkbox` why is that? Perhaps the second class may be `.checkbox-holder` What do you think?

Reply:
Yes, is an option @cbernat  may be ´.checkbox-holder´ is the nomenclature correct! Thanks 👍